src/main/kotlin/day17/mapper/PhotoMapper.kt

```
@@ -0,0 +1,37 @@
+fun PhotoMapper(dto: PhotoDto): DomainPhoto {
+    val type = when (dto.type?.lowercase(Locale.getDefault())) {
+        "article" -> PhotoType.Article
+        "image" -> PhotoType.Image
+        "video" -> PhotoType.Video
+        else -> PhotoType.Unknown
+    }
+
+    val createdAt = dto.createdAt?.let {// null이 아니면 let 블록이 실행
+        try {
+            LocalDate.parse(it, dateFormatter)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)